scripts/process_data.py

```diff
@@ -312,6 +312,10 @@ def main(self) -> None:  # pylint: disable=R0915
         image_dir = self.output_dir / "images"
         image_dir.mkdir(parents=True, exist_ok=True)
 
+        for i in self.crop_factor:
+            if i < 0 or i > 1:
+                raise ValueError("crop-factor values must be between 0 and 1")
+
```

Comment:
Add this check inside the functions that actually do the crop. Also for consistency, instead of using `raise`, do something like, `CONSOLE.print("[red] Invalid crop...` followed by a `sys.exit(1)`